configloader/configloader.go

```diff
@@ -163,13 +171,26 @@ func (l *Loader) loadFields(v reflect.Value, t reflect.Type, prefix string) erro
 			continue
 		}
 
-		if err := l.setFieldValue(field, l.env(envName)); err != nil {
+		if err := l.setFieldValue(field, l.getOrDefault(&fieldType, envName)); err != nil {
 			return fmt.Errorf("error setting field %s: %w", fieldType.Name, err)
 		}
 	}
 	return nil
 }
 
+func (l *Loader) getOrDefault(fieldType *reflect.StructField, envName string) string {
+	value, found := l.env(envName)
+	if found && value != "" {
+		return value
+	}
+
+	if value, found = fieldType.Tag.Lookup(l.defaultTag); found {
+		return value
+	}
+	// TODO: Error.
+	return value
```

Comment:
Add word approx if !found :D Levenshtein hehe

Reply:
Agree, let do something like this in another PR :D